resources/plugin/tables.go

```diff
@@ -117,127 +147,130 @@ func tables() []*schema.Table {
 		apprunner.VpcIngressConnections(),
 		appstream.AppBlocks(),
 		appstream.Applications(),
-		appstream.DirectoryConfigs(),
 		appstream.Fleets(),
 		appstream.ImageBuilders(),
 		appstream.Images(),
 		appstream.Stacks(),
-		appstream.UsageReportSubscriptions(),
 		appstream.Users(),
 		appsync.GraphqlApis(),
 		athena.DataCatalogs(),
 		athena.WorkGroups(),
-		autoscaling.LaunchConfigurations(),
+		auditmanager.Assessments(),
 		autoscaling.Groups(),
+		autoscaling.LaunchConfigurations(),
 		autoscaling.ScheduledActions(),
-		backup.GlobalSettings(),
 		backup.Plans(),
-		backup.RegionSettings(),
+		backup.ProtectedResources(),
+		backup.ReportPlans(),
 		backup.Vaults(),
+		batch.ComputeEnvironments(),
+		batch.JobDefinitions(),
+		batch.JobQueues(),
 		cloudformation.Stacks(),
+		cloudformation.StackSets(),
 		cloudfront.CachePolicies(),
 		cloudfront.Distributions(),
-		cloudhsmv2.Clusters(),
+		cloudfront.Functions(),
 		cloudhsmv2.Backups(),
+		cloudhsmv2.Clusters(),
+		cloudtrail.Channels(),
 		cloudtrail.Trails(),
-		cloudwatchlogs.ResourcePolicies(),
-		cloudwatchlogs.MetricFilters(),
-		cloudwatchlogs.LogGroups(),
 		cloudwatch.Alarms(),
+		cloudwatchlogs.LogGroups(),
+		codeartifact.Domains(),
+		codeartifact.Repositories(),
 		codebuild.Projects(),
-		codepipeline.Webhooks(),
+		codebuild.SourceCredentials(),
+		codecommit.Repositories(),
 		codepipeline.Pipelines(),
+		codepipeline.Webhooks(),
 		cognito.IdentityPools(),
 		cognito.UserPools(),
+		config.ConfigRules(),
+		config.ConfigurationAggregators(),
 		config.ConfigurationRecorders(),
 		config.ConformancePacks(),
-		config.ConfigRules(),
 		dax.Clusters(),
+		detective.Graphs(),
 		directconnect.Connections(),
 		directconnect.Gateways(),
 		directconnect.Lags(),
-		directconnect.VirtualGateways(),
 		directconnect.VirtualInterfaces(),
 		dms.ReplicationInstances(),
-		docdb.Clusters(),
-		docdb.ClusterParameterGroups(),
 		docdb.Certificates(),
-		docdb.EngineVersions(),
-		docdb.SubnetGroups(),
-		docdb.GlobalClusters(),
-		docdb.Events(),
+		docdb.ClusterParameterGroups(),
+		docdb.Clusters(),
 		docdb.EventSubscriptions(),
-		docdb.EventCategories(),
-		docdb.PendingMaintenanceActions(),
+		docdb.GlobalClusters(),
+		docdb.SubnetGroups(),
+		dynamodb.Backups(),
+		dynamodb.Exports(),
+		dynamodb.GlobalTables(),
 		dynamodb.Tables(),
-		ec2.ByoipCidrs(),
+		dynamodbstreams.Streams(),
+		ec2.CapacityReservations(),
 		ec2.CustomerGateways(),
 		ec2.EbsSnapshots(),
 		ec2.EbsVolumes(),
 		ec2.EgressOnlyInternetGateways(),
-		ec2.Eips(),
```

Comment:
Is this one of the things that was removed because it didn't have an ARN? If so, I think this one is a mistake (EIPs definitely have ARNs)